index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js">
+  <!--<![endif]-->
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Library</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    
+    <style>
+      div.container {
+        margin: 30px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container">
+    <!--[if lt IE 7]>
+      <p class="browsehappy">
+        You are using an <strong>outdated</strong> browser. Please
+        <a href="#">upgrade your browser</a> to improve your experience.
+      </p>
+    <![endif]-->
+    <form id="myForm">
+      <div class="form-group">
+      <label for="title">Title</label>
+      <input class="form-control" type="text" id="title">
+      </div>
+
+      <div class="form-group">
+      <label for="author">Author</label>
+      <input class="form-control" type="text" id="author-name">
+      </div>
+
+      <div class="form-group">
+      <label for="pages">Number of Pages</label>
+      <input class="form-control" type="number" id="pages">
+      </div>
+      <div class="form-select">
+      <label for="read">Read Yet?</label>
+      <select name="read" id="read">
+        <option value="true">Yes</option>
+        <option value="false">No</option>
+      </select>
+      </div>
+      <button class="btn btn-primary" onclick="addToLibrary()">Add Book To Library</button>
+
+    </form>
+
+    <br>
+
+    <table class="table table-striped">
+      <tr>
+    <th>Title</th>
+    <th>Author</th>
+    <th>Number of Pages</th>
+    <th>Read?</th>
```

Comment:
Headers for the `Mark as read` and `remove` buttons are missing and when you add records to the table the header looks weird